.mk/k8s.mk

```diff
@@ -36,7 +36,7 @@ endif
 DEPLOY_NSM = nsmgr $(DEPLOY_FORWARDING_PLANE)
 DEPLOY_PROXY_NSM = proxy-nsmgr
 DEPLOY_INFRA = $(DEPLOY_TRACING) $(DEPLOY_WEBHOOK) $(DEPLOY_NSM) $(DEPLOY_PROXY_NSM) $(DEPLOY_MONITOR)
-DEPLOYS = $(DEPLOY_INFRA) $(DEPLOY_ICMP) $(DEPLOY_VPN)
+DEPLOYS = $(DEPLOY_INFRA) $(DEPLOY_ICMP) $(DEPLOY_VPN) nsm-coredns nsm-monitor
```

Comment:
Do you think it'd be better if it is part of DEPLOY_NSM or a separate DEPLOY variable? I'm saying that because if someone deploys only the infra part, then these two will not be deployed/loaded.

Reply:
actually `DEPLOY_INFRA` used only to collect final list of DEPLOYS. But we can re-use these variables in the future. Fixed.